float64 x
float64 y
string arithimetic_operator 
time timestamp
---
float64 outcome
float64 with_param_factor
string my_service_name
string client_name
time network_time
